cmd/main.go

```diff
@@ -162,21 +163,30 @@ func main() {
 		}
 	}
 
+	Options.InstructionConfig.ReleaseImageMirror = Options.ReleaseImageMirror
+	Options.JobConfig.ReleaseImageMirror = Options.ReleaseImageMirror
```

Comment:
Not for this PR, but do we have a techdebt issue to track removing (or renaming) all of this `Job` stuff as we don't have any actual `Jobs` anymore?

Reply:
@filanov

Reply:
Please open a ticket.